app/unitPages.rb

```diff
@@ -632,6 +634,7 @@ def getUnitProfile(unit, attrs)
   
   attrs['directSubmit'] and profile[:directSubmit] = attrs['directSubmit']
   attrs['directSubmitURL'] and profile[:directSubmitURL] = attrs['directSubmitURL']
+  attrs['directManageURL'] and profile[:directManageURL] = attrs['directManageURL']
```

Comment:
Do we need directManage attrs in addition to directManageURL here?

Reply:
not sure, I was copying the directSubmitURL code